hw/i386/acpi-build.c

```diff
@@ -1214,41 +1057,10 @@ static void build_piix4_isa_bridge(Aml *table)
 static void build_piix4_pci_hotplug(Aml *table)
 {
     Aml *scope;
```

Comment:
Where does the logic that went here for PIIX reside now? build_acpi_pci_hotplug? Did all of the logic here move to that function.

Reply:
build_piix4_pci_hotplug calls build_acpi_pci_hotplug. ?Yes, I think all the logic here are moved into the new function.